xapian-core/include/xapian/mset.h

```diff
@@ -47,6 +48,9 @@ class XAPIAN_VISIBILITY_DEFAULT MSet {
     // Helper function for fetch() methods.
     void fetch_(Xapian::doccount first, Xapian::doccount last) const;
 
+    // Helper function for replace_weights(Iterator first, Iterator last)
```

Comment:
We should probably say "See the note about correct usage in the docstring for `Xapian::Internal::MSet::set_item_weight()`" or something similar.